doc/logging.md

```diff
@@ -0,0 +1,24 @@
+# How does Happy test log work
+
+## Types of Happy logs
+  * Happy host logs — Happy will run as host, Happy host logs will have logs from Happy tool only.
+
+  * Happy device logs — Happy will host virtual network which has multiple virtual devices, Happy device logs will have logs for software running on the virtual device, openweave is one of the software example, so device logs will have openweave logs.
+
+## Where Happy logs are stored
+
+Both Happy host and device logs location are processed in `happy/Driver.py` as below:
+  * If `HAPPY_LOG_DIR` is found in os environment variable, Happy uses `HAPPY_LOG_DIR` value as log folder location to store Happy test logs. For example, openweave-core tests store Happy test logs along with feature folder under build folder.
+  * If `HAPPY_LOG_DIR` is not found in os environment variable, Happy uses `default_happy_log_dir` from [happy/conf/main_config.json](https://github.com/openweave/happy/blob/master/happy/conf/main_config.json) as the default log folder location to store Happy test logs.
+
+## How to set `HAPPY_LOG_DIR`
+  There is multiple ways to set `HAPPY_LOG_DIR` in os environment variable, see examples below:
+  * In openweave test, we add `HAPPY_LOG_DIR=<dir value>` in command line along with the test script we want to run in [weave-parallel-test-engine](https://github.com/openweave/openweave-core/blob/master/src/test-apps/weave-parallel-test-engine)
```

Comment:
This doesn't belong here. The Happy package shouldn't be aware of how packages that depend on it are using it. Consequently, it shouldn't link to the openweave-core project either.

A better and less-coupled example might be: "If your project is being built with GNU autotools and you are running functional and unit tests with Happy with the DejaGnu test framework, then you might add HAPPY_LOG_DIR=<dir value> to TESTS_ENVIRONMENT in your Makefile.am or Makefile.in.

Reply:
sure, I will remove how openweave-core tests using this variable.